### PR TITLE
Notes for 4th January meeting

### PR DESCRIPTION
**Pull request by cmarmo**
I have copied - pasted the notes.
It could be useful to re-read once, before merge just to be sure that your future selfs will be able to understand.

### CODE REVIEW

File: meeting_notes/2021-01-04.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+         - Adrin to add references and/or sample code
+
+### General topics
+
+- Loïc: enable Github Discussions? One hope would be to help getting people involved outside the core developers.
+  [Github Discussions doc](https://docs.github.com/en/free-pro-team@latest/discussions)
+    - Reshama: how many people are on the mailing list? over 2k
+    - + Discussions would allow for better coding formatting
+    - We should be able to enforce our CoC
+    - We would need to extend the triage team who can set the accepted answers and make sure the answers are indeed correct
```
[the remark this answers is not in the document]
Comment (rth) [reply meeting_notes/2021-01-04.md:66]: cc @glemaitre

**Comment by rth** [comment]
Thanks for keeping these notes! They are really helpful for people who miss the meetings.

**Comment by cmarmo** [comment]
> Thanks for keeping these notes! They are really helpful for people who miss the meetings.

cc @adrinjalali :)

File: meeting_notes/2021-01-04.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (thomasjpfan) [reply meeting_notes/2021-01-04.md:66]: This is in the context of GitHub discussions. GitHub discussions has a "accept answers" feature. If there are users using GitHub discussions, we would need more moderators/triage members to look after it.

**Comment by cmarmo** [comment]
I'm tempting the gods and merging this one.

**Pull request merged**